File: DoAnOOP/NhanVien.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DoAnOOP
{
   public class NhanVien
    {
        public string MSNV { get; set; }
        public string HoTenNV { get; set; }
        public string DiaChi { get; set; }
        public DateTime NgaySinh { get; set; }
        public bool GioiTinh { get; set; }
        public string SDT { get; set; }
        public string Email { get; set; }

        public NhanVien(string MSNV, string HoTenNV, string DiaChi, DateTime NgaySinh, bool GioiTinh, string SDT, string Email)
        {
            this.MSNV = MSNV;
            this.HoTenNV = HoTenNV;
            this.DiaChi = DiaChi;
            this.NgaySinh = NgaySinh;
            this.GioiTinh = GioiTinh;
            this.SDT = SDT;
            this.Email = Email;
        }
        public NhanVien(NhanVien nv)
        {
            this.MSNV = nv.MSNV;
            this.HoTenNV = nv.HoTenNV;
            this.DiaChi = nv.DiaChi;
            this.NgaySinh = nv.NgaySinh;
            this.GioiTinh = nv.GioiTinh;
            this.SDT = nv.SDT;
            this.Email = nv.Email;
        }

    }
}

File: java/cuoiki/src/mypack/NHANVIEN.java
package mypack;

import java.util.Date;

public class NHANVIEN {
	public String MSNV;
	public String HoTenNV;
	public String DiaChi;
	public Date NgaySinh;
	public boolean GioiTinh;
	public String SDT;
	public String Email;
	public NHANVIEN(String mSNV, String hoTenNV, String diaChi, Date ngaySinh, boolean gioiTinh, String sDT,
			String email) {
		super();
		MSNV = mSNV;
		HoTenNV = hoTenNV;
		DiaChi = diaChi;
		NgaySinh = ngaySinh;
		GioiTinh = gioiTinh;
		SDT = sDT;
		Email = email;
	}
	public String getMSNV() {
		return MSNV;
	}
	public void setMSNV(String mSNV) {
		MSNV = mSNV;
	}
	public String getHoTenNV() {
		return HoTenNV;
	}
	public void setHoTenNV(String hoTenNV) {
		HoTenNV = hoTenNV;
	}
	public String getDiaChi() {
		return DiaChi;
	}
	public void setDiaChi(String diaChi) {
		DiaChi = diaChi;
	}
	public Date getNgaySinh() {
		return NgaySinh;
	}
	public void setNgaySinh(Date ngaySinh) {
		NgaySinh = ngaySinh;
	}
	public boolean isGioiTinh() {
		return GioiTinh;
	}
	public void setGioiTinh(boolean gioiTinh) {
		GioiTinh = gioiTinh;
	}
	public String getSDT() {
		return SDT;
	}
	public void setSDT(String sDT) {
		SDT = sDT;
	}
	public String getEmail() {
		return Email;
	}
	public void setEmail(String email) {
		Email = email;
	}
	
}
